Client/Frontend/Browser/ReaddleSparkIntegration.swift

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+struct ReaddleSparkIntegration {
+    static let supportedHeaders = [
+        "recipient",
+        "html",
+        "subject",
+        "textbody"
+    ]
+
+    static func newEmailURLFromMetadata(metadata: MailToMetadata) -> NSURL {
+        var readdleSparkMailURL = "readdle-spark://compose?"
+
+        // The web is a crazy place and some people like to capitalize the hname values in their mailto: links.
+        // Make sure we lowercase anything we found in the metadata since Outlook requires them to be lower case.
```

Comment:
Update the comment as it references Outlook. Also, I can see this logic being in other newEmailURL... handlers so it might be useful to pull it out into it's own method.